proto/integration.proto

```diff
@@ -447,4 +447,12 @@ message ForkresolutionConfigRequest{
 
 service ForkresolutionService{
     rpc CompareTwoBranches(TwoBranchesRequest) returns (CompareReply) {}
+}
+
+message EmptyReply{
+}
+
+service PopService{
+    rpc CheckVTBInternally(BytesArrayRequest) returns (EmptyReply) {}
```

Comment:
Does it make sense to return empty reply?

Reply:
it should return bool, isn't it?